extensions/applicationinsights-analytics-js/typedoc.json

```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "https://typedoc.org/schema.json",
+    "entryPoints": [ "./src/applicationinsights-analytics-js.ts" ],
+    "exclude": [ "**/internal/**/*.ts", "node_modules/**" ],
+    "externalPattern": [
+        "**/node_modules/**",
+        "node_modules/**"
+    ],
+    "sort": [
+        "alphabetical",
+        "kind",
+        "instance-first"
+    ],
+    "basePath": "./src",
+    "sourceLinkTemplate": "https://github.com/microsoft/ApplicationInsights-JS/blob/main/{path}#L{line}",
+    "cleanOutputDir": true,
+    "excludeExternals": false,
+    "excludeInternal": true,
+    "excludePrivate": true,
+    "includeVersion": true,
+    "groupOrder": [
+        "Entrypoint",
+        "Modules",
+        "Namespaces",
+        "Enumerations",
+        "Enumeration Members",
+        "Classes",
+        "Interfaces",
+        "Type Aliases",
+        "Constructors",
+        "Properties",
+        "Variables",
+        "Functions",
+        "Accessors",
+        "Methods",
+        "References",
+        "*"
+    ],
+    "tsconfig": "./tsconfig.json",
+    "out": "../../docs/extensions/applicationinsights-analytics-js",
```

Comment:
These should be
`../../docs/webSdk/<packageName>`

so for this one
`../../docs/webSdk/applicationinsights-analytics-js`

You will need to "cleanup" all of these that are generated in the wrong location.

Reply:
ok, so all of the docs would go to docs/webSdk folder.